### ISSUE TITLE
[Feature Request] Main Dialog - Open in Custom URL

### ISSUE THREAD

**Issue by FelipeRearden** [tadashi-aikawa/obsidian-another-quick-switcher#213]
Hello @tadashi-aikawa !!!!

This FR is a variation of the feature `open in google`.

Instead of opening the content from the `input query` in a google search my FR is about let the user customize an URL to use the `input query` 

Let me show the workflows 

## 1️⃣ Search inside the Vault

I would like to start a vault search from the `input query` from an AQS Modal.

`{{iq}}` means `input query`

I can build an url like this

```
obsidian://search?vault=AAA&query=content:"{{iq}}"
```

📸 to open the Obsidian native search from an AQS Modal !!!!!
![IMG_1102](https://github.com/tadashi-aikawa/obsidian-another-quick-switcher/assets/85180696/7bd49431-bcde-4cf6-b99c-f4589db3bc78)

**Additional motivation for this: this integration between AQS and Obsidian native search could be an easy way to be able to search ALL vault from AQS !!!!!**

## 2️⃣ Open Apps outside Obsidian (URL scheme)

As you know I use Obsidian Mobile on iPad as well and I use the iPadOS feature **Slide View** often from Obsidian to open apps in slide view using the templater.

One example I wanna perform an search using the `input query` from an AQS Modal using the app ORION which supports URL SCHEME

I can build an url like this

```
orion://search?q={{iq}}
```

📸 to open orion on slide view from an AQS Modal !!!!!
![IMG_1103](https://github.com/tadashi-aikawa/obsidian-another-quick-switcher/assets/85180696/2d6c9848-84b6-4cf5-ab23-2c03afed1a5b)

---

I have a bunch of examples, but I choose these two because it shows options outside Obsidian and for ipad users 

---

Thanks for reading this !!!!! I hope you like my suggestion 🙏 🙏 🙏

Let me know if you need more information !!!

Have a great day 🌞

**Comment by tadashi-aikawa** [comment]
Thank you for your FR, @FelipeRearden :)

That's a nice idea! I'll consider implementing it 👍

**Comment by FelipeRearden** [comment]
> Thank you for your FR, @FelipeRearden :)
> 
> That's a nice idea! I'll consider implementing it 👍

Hello @tadashi-aikawa !!!!!

Thank you very much for the reply !!!!

Just one more idea about this to you to decide when implement this FR is the future. If possible 🙏

1️⃣ more than one URL Custom Field: in my opinion, 3 would be a good number 

2️⃣ different configuration for Obsidian mobile. Since there are apps exclusive for mobile would be better to have the option to have different configuration based on the Desktop and Mobile.

**Comment by tadashi-aikawa** [comment]
@FelipeRearden 
I agree with your idea. However, this feature is not a primary focus of Another Quick Switcher, so initially, I'll replace the Google search with a custom schema.

**Comment by FelipeRearden** [comment]
@tadashi-aikawa 
100% understood! 

Thank you very much for the custom schema around Google search and for considering more custom schema for possible future releases !
